Update install.bat

simon-visser opened the pull request:
Added -p to mkdir so that make directory even if exists

simon-visser committed:
Update install.bat

Added -p to mkdir so that make directory even if exists

KevinHelfrich commented on install.bat:
```diff
@@ -1,4 +1,4 @@
-mkdir C:\securityTrainingScript
+mkdir -p C:\securityTrainingScript
```
Can you point me too something showing this works on windows.

-p seems to be a linux only flag from what I can find on google, and my own testing.

The rest of the code in the repo is windows specific, so any fix has to work on windows.

KevinHelfrich replied:
Maybe try something along the lines of:

https://stackoverflow.com/questions/4165387/create-folder-with-batch-but-only-if-it-doesnt-already-exist

simon-visser committed:
Correctly handling 'directory already exists'

Solution pulled from: 

https://stackoverflow.com/questions/4165387/create-folder-with-batch-but-only-if-it-doesnt-already-exist

(-p is a unix command)

simon-visser commented:
Quite right, I've changed the commit to reflect your suggestions. That SO answer is perfect, and solution is tested.

KevinHelfrich commented:
Thanks. Verified it as well. Appears to work. Merged it.
